HelpSource/Classes/Server.schelp

```diff
@@ -631,6 +631,13 @@ Show frequency analyzer window.
 method:: meter
 Show input/output level meters.
 
+argument:: numIns
+the number of input channels. The default is nil and the number of input channels defined by code::Server.local.options.numInputBusChannels:: is used when using a local server.
+argument:: numOuts
+the number of output channels. The default is nil and the number of output channels defined by code::Server.local.options.numOutputBusChannels:: is used when using a local server.
+argument:: position
+the position of the scope window. The value should be a point: x@y. The default is nil and the scope window will be in the centre of the display.
```

Comment:
I'd suggest capitalizing "The" at the beginning of each sentence.

> The value should be a point: x@y

As implemented the more accurate description would be:  

The value should be a point, teletype::x@y::, or respond to teletype::asPoint::.

Also consider using teletype:: or code:: when writing `nil`, as it's code.

Reply:
Thanks! Done in https://github.com/supercollider/supercollider/pull/6283/commits/0b0936e8cb1a4c201f4cd91d2876c6202619f953

I did not capitalised the `the` in the following:
>the position of the scope window. 

If you think I should capalise it as well, please let me know.